contrib/babelfishpg_tds/src/include/tds_iofuncmap.h

```diff
@@ -49,6 +49,8 @@
 #define		TDS_SEND_XML					26
 #define		TDS_SEND_SQLVARIANT				28
 #define		TDS_SEND_DATETIMEOFFSET				29
+#define     TDS_SEND_GEOMETRY               30
+#define     TDS_SEND_GEOGRAPHY              31
```

Comment:
Nit: indentation seems a bit off. This comment also applies to all the changes in this file.

Reply:
Resolved